shopcart.py

```diff
@@ -138,7 +138,7 @@ def get_shopcart(sid):
 ######################################################################
 # RETRIEVE USER'S PRODUCTS LIST IN THE CART
 ######################################################################
-# USAGE: /shopcarts/3/products or /shopcarts/3/products?sku=114672050 for querying for a product
+# USAGE: /shopcarts/3/products or /shopcarts/3/products?name=Settlers of Catan for querying for a product
```

Comment:
We can even query by sku right? so shouldnt we keep both? @WesPainter @jiweix @ceciliazhang12